index.html

```diff
@@ -2909,6 +2909,19 @@ <h2>Key and Signature Expiration</h2>
       </p>
     </section>
 
+    <section>
+      <h3>Verification Method Binding</h3>
+
+      <p>
+Implementers ensure that a [=verification method=] is bound to a particular
+[=controller=] by going from the expression of the [=verification method=] to
+the [=controller document=], and then ensuring that the [=controller document=]
+also contains a reference to the [=verification method=]. This process is
+described in the algorithm for <a href="#retrieve-verification-method">
+retrieving a verification method</a>.
```

Comment:
Just syncing up with the similar discussion in another thread:

>> Note that the additional property I propose (linking the VM to the CD) is absolutely optional and does not contradict with what you say; you also talk in conditional. The only reason I think the property would be useful is because I presume it would simplify a number of business rules. But, again, if we decide it is too late for that, be it.
>
> Yeah, I do believe it's too late for that but it could perhaps be further debated/bikeshedded/sorted and tested in a future minor release. But it's not too late for us to add some non-normative text somewhere as needed.

(see https://github.com/w3c/controller-document/issues/119#issuecomment-2486285552).